src/fe/inf_fe_map.C

```diff
@@ -486,17 +486,18 @@ Point InfFE<Dim,T_radial,T_map>::inverse_map (const Elem * inf_elem,
           p(2) = 1. - 2*a_interpolated/fp_o_dist;
 
 #ifdef DEBUG
-
+  /*
 
           // the radial distance should always be >= -1.
 
-          // if (p(2)+1 < tolerance)
-          //  {
-          // libmesh_here();
-          // libMesh::err << "WARNING: radial distance p(2) is "
-          //      << p(2)
-          //      << std::endl;
-          //  }
+           if (p(2)+1 < tolerance)
+            {
+           libmesh_here();
+           libMesh::err << "WARNING: radial distance p(2) is "
+                << p(2)
+                << std::endl;
+            }
+          */
 #endif
```

Comment:
Please either undo the commenting change or simply delete the code altogether.  There's no reason for this to be part of the diff...